docs/int/quickstart/advanced/quickstart-builder-api.md

```diff
@@ -43,41 +45,119 @@ Charon supports builder API with the `--builder-api` flag. To use builder API, o
+      </code>
+    </pre>
+    Or you can configure it to communicate with a local <a href="https://github.com/flashbots/mev-boost" target="_blank">MEV-boost</a> sidecar to configure multiple relays:
+    <pre>
+      <code>
+    {String.raw`lighthouse bn --builder "http://mev-boost:18550"`}
+      </code>
+    </pre>
+  </TabItem>
+  <TabItem value="prysm" label="Prysm">
+    <pre>
+      <code>
+    {String.raw`prysm beacon-chain --http-mev-relay "https://0xac6e77dfe25ecd6110b8e780608cce0dab71fdd5ebea22a16c0205200f2f8e2e3ad3b71d3499c54ad14d6c21b41a37ae@boost-relay.flashbots.net"`}
+      </code>
+    </pre>
+  </TabItem>
+  <TabItem value="nimbus" label="Nimbus">
+    <pre>
+      <code>
+        {String.raw`--payload-builder=true --payload-builder-url="https://0xac6e77dfe25ecd6110b8e780608cce0dab71fdd5ebea22a16c0205200f2f8e2e3ad3b71d3499c54ad14d6c21b41a37ae@boost-relay.flashbots.net"`}
+      </code>
+    </pre>
+    You should also consider adding <code>--local-block-value-boost 3</code> as a flag, to favour locally built blocks if they are withing 3% in value of the relay block, to improve the chances of a successful proposal.
+  </TabItem>
+  <TabItem value="lodestar" label="Lodestar">
+    <pre>
+      <code>
+    {String.raw`--builder --builder.urls "https://0xac6e77dfe25ecd6110b8e780608cce0dab71fdd5ebea22a16c0205200f2f8e2e3ad3b71d3499c54ad14d6c21b41a37ae@boost-relay.flashbots.net"`}
+      </code>
+    </pre>
+  </TabItem>
+</Tabs>
 
-Once the flag is set up, Obol distributed validators will be able to register to the builder network, submit blinded beacon blocks and gain a share of the MEV profits.
-
-#### Lighthouse Validator Client
-
-For lighthouse, configuring the VC with Charon can be done by following the same process outlined in the [Lighthouse official guide](https://lighthouse-book.sigmaprime.io/builders.html).
-
-The validator client must be set up to use the `--builder-proposals` [flag](https://lighthouse-book.sigmaprime.io/builders.html#how-to-connect-to-a-builder) with a value equal to `http://$CHARON_ENDPOINT:3600/proposer_config`.
+### Validator Clients
 
-Once the flag is set up, Obol distributed validators will be able to register to the builder network, submit blinded beacon blocks and gain a share of the MEV profits.
+The following flags need to be configured on your chosen validator client
+
+<Tabs groupId="validator-clients">
+  <TabItem value="teku" label="Teku" default>
+    <pre>
+      <code>
+    {String.raw`teku validator-client --validators-proposer-blinded-blocks-enabled=true `}
+      </code>
+    </pre>
+
+  </TabItem>
+  <TabItem value="lighthouse" label="Lighthouse">
+    <pre>
+      <code>
+    {String.raw`lighthouse vc --builder-proposals`}
+      </code>
+    </pre>
+  </TabItem>
+  <TabItem value="prysm" label="Prysm">
+    <pre>
+      <code>
+    {String.raw`prysm validator --enable-builder`}
```

Comment:
also might be good to mention` --suggested-fee-recipient=0xf6f9eaE9FF22219B933744049EB89f2f0da77baC`

Reply:
Have you experienced issues without this? because I fear that it will result in people hardcoding the wrong fee recipient and missing proposals if they don't change it to their correct one? Not implementing this change for now, but lmk.